apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/MoreActions/NotesDialog.tsx

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { FormProvider, SubmitHandler, useForm } from "react-hook-form";
+import ChatBubbleBottomCenterIcon from "@heroicons/react/24/outline/ChatBubbleBottomCenterIcon";
+import { useMutation } from "urql";
+
+import { Button, Dialog } from "@gc-digital-talent/ui";
+import { Submit, TextArea } from "@gc-digital-talent/forms";
+import { Maybe, PoolCandidate, graphql } from "@gc-digital-talent/graphql";
+import { toast } from "@gc-digital-talent/toast";
+import { errorMessages, formMessages } from "@gc-digital-talent/i18n";
+
+type FormValues = {
+  notes?: PoolCandidate["notes"];
+};
+
+interface NotesDialogProps {
+  poolCandidateId: string;
+  notes: Maybe<string> | undefined;
+}
+
+const PoolCandidate_UpdateMutation = graphql(/* GraphQL */ `
+  mutation UpdatePoolCandidate(
+    $id: ID!
+    $input: UpdatePoolCandidateAsAdminInput!
+  ) {
+    updatePoolCandidateAsAdmin(id: $id, poolCandidate: $input) {
+      id
+      notes
+    }
+  }
+`);
```

Comment:
This mutation name is already in use and causing a duplicate identifier error. We will need to namespace it. I suggest `PoolCandidate_NotesUpdate` or something along those lines.

Reply:
Okay, should be good to go now [b7ef9aa](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8853/commits/b7ef9aa821fd4fe1b8d9bb92b5d128a6202550c6)